docs/api-data.md

```diff
@@ -0,0 +1,39 @@
+we get a bunch of json data from the drupal-powered cms using the jsonapi module and friends.
```

Comment:
Thanks for adding this. Need to start building out more developer documentation and this is a good addition.